data/human/hails.txt

```diff
@@ -2082,6 +2082,79 @@ phrase "friendly civilian"
 		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
 		"New Holland. It gets too boring there, though, so I fly the stars instead."
 		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
+		
+phrase "pyramid scheme"
+	word
+		"Magellan "
+		"Dehydrated "
+		"Pontificated "
+		"General "
+		"Sex Sex Sex "
```

Comment:
```suggestion
```

Reply:
This suggestion didn't get committed if you wanted it to be, it was only resolved.